package.json

```diff
@@ -200,6 +202,7 @@
     "@types/semver": "^7.3.4",
     "@types/serve-static": "^1.13.8",
     "@types/shelljs": "^0.8.7",
+    "@types/terser-webpack-plugin": "^4.2.0",
```

Comment:
The monorepo root should get the version 4 of webpack & it's sibling dependencies! It wasn't... I could tweak the lockfile, but rather just instruct yarn that we want to install v4 in the root. This does that trick.